test/calico/metadata_access_test.go

```diff
@@ -57,7 +57,7 @@ spec:
 apiVersion: v1
 kind: Namespace
 metadata:
-  generateName: metadata-access-test-
+  generateName: test-
```

Comment:
Having an unique indentifier on the namespace name is IMO valuable, as it makes it easier to find the right namespace to inspect when test fails.

I'd either remove this field or leave a placeholder here and then set it programatically, using something like this:
```go
ns.Metadata.GenerateName = util.TestNamespace("metadata-access") // should return "test-metadata-access-"
```